concordia/templates/action-app.html

```diff
@@ -14,6 +14,7 @@
     <script type="module">
         import { ActionApp } from "{% static 'js/action-app/index.js' %}";
         window.actionApp = new ActionApp({
+            currentUser: "{{ request.user.pk }}",
```

Comment:
This should behave like a normal JavaScript variable and either be undefined or the value so checks don't need to guard against the “None” string

Reply:
Done